components/openzwave-adapter/src/lib.rs

```diff
@@ -182,8 +182,8 @@ pub struct OpenzwaveAdapter {
     version: [u32; 4],
     ozw: ZWaveManager,
     node_map: IdMap<ServiceId, Node>,
-    getter_map: IdMap<Getter, ValueID>,
-    setter_map: IdMap<Setter, ValueID>,
+    getter_map: IdMap<Channel, ValueID>,
+    setter_map: IdMap<Channel, ValueID>,
```

Comment:
I'm currently writing the patch to do this in Taxonomy, so yeah, that's planned.